### ISSUE TITLE
Auth48: Combined internal references

### ISSUE THREAD

**Issue by MikeBishop** [quicwg/base-drafts#4955]
https://github.com/cabo/kramdown-rfc2629 supports references to other documents of the form `{{Sections X.Y and X.Z of BLAHBLAH}}`, which renders as one might expect. In multiple locations, the draft makes multiple internal references (`see {{foo}} and {{bar}}`) which render as "see Section X.Y and Section X.Z".

The RFC Editor asks that these be changed to "Sections X.Y and X.Z", but I'm currently unaware of a way to do that within a document in kramdown.

@martinthomson, @cabo - suggestions appreciated.

**Comment by cabo** [comment]
You can use the format= attribute of <xref.

``` 
{{<terminology}} for <xref target="terminology" format="counter"/>
{{<<terminology}} for <xref target="terminology" format="title"/>
``` 

So your example becomes

```
Sections {{<foo}} and {{<bar}}
```

Enjoy!

**Comment by MikeBishop** [comment]
Thank you!  That trick could be useful in a few other places as well.

**Comment by MikeBishop** [comment]
Closed in #4956.
